frontend/src/e2e-playwright/pages/mobile/mobile-nav.ts

```diff
@@ -0,0 +1,53 @@
+// SPDX-FileCopyrightText: 2017-2020 City of Espoo
```

Comment:
Supernit: should be 2021 (the helper script should have the correct date but I assume this is from your IDE?)

Reply:
Fixed! Copy-pasted from some other file :see_no_evil: